src/Microsoft.AspNet.Mvc.TagHelpers/FormTagHelper.cs

```diff
@@ -124,6 +125,14 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
             }
 
             var antiforgeryDefault = true;
+            if (output.Attributes.ContainsName("method"))
```

Comment:
> That way we can just pass it through to the `HtmlGenerator`

Note passing it into the `IHtmlGenerator` isn't useful per se and `FormMethod.Get.ToString()` will still be necessary. `GenerateForm()` doesn't deal with the antiforgery token but would need the `method` value to add the bound attribute back. And we need the `string` comparison to determine the `antiforgeryDefault` value.